messaging-experiments-82.yaml

```diff
@@ -237,7 +237,7 @@
   enabled: true
   # The filter `!('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)` is used to exclude
   # users who have switched channels or re-installed the browser without having a new profile
-  targeting: firefoxVersion >= '82.' && browserSettings.update.channel == 'beta' && localeLanguageCode == "en" && ((isFirstStartup && !('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)) || 'bug-1668928-message-multi-stage-aboutwelcome-zero-stage-release-82-83' in activeExperiments)
+  targeting: firefoxVersion >= '82.' && browserSettings.update.channel != 'release' && localeLanguageCode == "en" && ((isFirstStartup && !('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)) || 'bug-1668928-message-multi-stage-aboutwelcome-zero-stage-release-82-83' in activeExperiments)
```

Comment:
Looks like we need to remove people who opt out:
https://github.com/mozilla/messaging-system-inflight-assets/blob/79345e8aacef3f4c544489f720c9bcae9f1fff5c/messaging-experiments-82.yaml#L235